src/libraries/Microsoft.Extensions.Hosting/src/Internal/HostingLoggerExtensions.cs

```diff
@@ -87,5 +86,17 @@ public static void BackgroundServiceFaulted(this ILogger logger, Exception ex)
                     message: "BackgroundService failed");
             }
         }
+
+        public static void BackgroundServiceStoppingHost(this ILogger logger, Exception ex)
+        {
+            if (logger.IsEnabled(LogLevel.Critical))
+            {
+                logger.LogCritical(
+                    eventId: LoggerEventIds.BackgroundServiceStoppingHost,
+                    exception: ex,
+                    message: SR.BackgroundServiceExceptionStoppedHost,
+                    args: ex);
```

Comment:
I think this `args: ex` can be removed now, since the exception is flown into `exception`.

Reply:
Ah, yes - good catch. So, `args` is if there is formatting expectations in the `message` - right?